pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -161,16 +163,25 @@ def select(fn):
             return False
         return not is_blacklist(fn)
 
+    # cleaning function (for reproducible builds)
+    def clean(tinfo):
```

Comment:
Nitpick: I see we didn't do it for the other inner helper function above, but I would use a docstring here, because that's what it's for.
```suggestion
    def clean(tinfo):
        """cleaning function (for reproducible builds)"""
```